Update triggering-pipelines.md (#4709)
harness/developer-hub: docs/platform/triggers/triggering-pipelines.md

<!-- this section is referenced on triggers-reference.md -->

Currently, CI pipeline webhook triggers don't support PRs that are attempting to merge changes from a Bitbucket forked repo into the original, base repo. This applies only to Bitbucket repos when attempting to merge a fork back into the base repo and the base repo is set as the pipeline's codebase. Although the trigger initiates, the pipeline fails with `couldn't find remote ref`. This issue occurs due to the Bitbucket PR reference URL format.

> ### Pipeline with two triggers has a PR Event Trigger and On Push Trigger and it fires the pipeline execution twice

Reviewer: ```suggestion
### Pipeline with a PR Event Trigger and an On Push Trigger fires the pipeline execution twice
```

> ### Pipeline with two triggers has a PR Event Trigger and On Push Trigger and it's firing the Pipeline execution twice.

Reviewer: ```suggestion
### Pipeline with two triggers (a PR Event Trigger and an On Push Trigger) fires the pipeline execution twice
```

> ### Pipeline with two triggers has a PR Event Trigger and On Push Trigger and it's firing the Pipeline execution twice.

Reviewer: ```suggestion
### Pipeline with two triggers has a PR Event Trigger and On Push Trigger and it fires the pipeline execution twice
```

> You can configure multiple triggers for the same pipeline. By scoping the action events, you can ensure that the Pipeline only runs for a particular trigger scenario. The Push trigger and PR Trigger can overlap because they listen on similar events. Below are the required events for PR to ensure there is no overlap with the Push Trigger.

Reviewer: ```suggestion
You can configure multiple triggers for the same pipeline. By scoping the action events, you can ensure that the pipeline only runs for a particular trigger scenario. The Push trigger and PR trigger can overlap because they listen on similar events. Below are the required events for the PR to ensure there is no overlap with the Push trigger.
```

**Pull Request Actions**
- Edit
- Open
- ReOpen
- Label
- Unlabel
- Ready for Review

> Please leave out `synchronize` and `close` from the event selection. These two events are what causes both triggers to execute the pipeline. 

Reviewer: ```suggestion
Leave out by the `synchronize` and `close` events from the event selection. These events cause both triggers to execute the pipeline. 
```
